librz/core/cil.c

```diff
@@ -127,7 +131,7 @@ static char *get_esil_stack_name(RzCore *core, const char *name, ut64 *addr, ut3
  * \param addr Base address of the stack region, if UT64_MAX it is automatically computed
  * \param size Size of the stack region, if UT32_MAX it is automatically computed
  */
-RZ_IPI void rz_core_analysis_esil_init_mem(RzCore *core, const char *name, ut64 addr, ut32 size) {
+RZ_API void rz_core_analysis_esil_init_mem(RzCore *core, RZ_NULLABLE const char *name, ut64 addr, ut32 size) {
 	ut64 current_offset = core->offset;
```

Comment:
add checks and rz_nonnull

Reply:
https://github.com/rizinorg/rizin/blob/2b9e68233f1cc0d6c12dcdc50f3a982d2a7173e7/librz/core/cil.c#L129-L130
https://github.com/rizinorg/rizin/blob/2b9e68233f1cc0d6c12dcdc50f3a982d2a7173e7/librz/core/cmd/cmd_analysis.c#L1879-L1880

it should be nullable

Reply:
no, i'm talking about RzCore, not name.

Reply:
OK